src/steps.tsx

```diff
@@ -253,8 +249,15 @@ const riskAssessment: Step[] = [
   },
   {
     id: 'askTraveledAffectedAreas',
-    message:
-      'Have you traveled to any of these areas in the past 14 days: Mainland China, Hong Kong, Iran, Italy, Japan, South Korea, Singapore',
+    message: `
+    Have you traveled to **any** of these areas in the past 14 days: 
```

Comment:
shouldn't this be _emphasized_ as opposed to **bolded**?

Reply:
I think that it stands out more if we bold it